assets/js/modules/sign-in-with-google/components/settings/SettingsEdit.stories.js

```diff
@@ -97,11 +114,16 @@ InvalidClientID.decorators = [
 	},
 ];
 
+export const Empty = Template.bind( null );
+Empty.storyName = 'Empty';
+Empty.scenario = {};
```

Comment:
I don't really think this one needs a VRT either, as the extra notice is additive and it's fine to just have one VRT for all elements on the screen.